cmd/agent/install_script.sh

```diff
@@ -208,12 +207,24 @@ elif [ $OS = "SUSE" ]; then
       ARCHI="x86_64"
   fi
 
+  # Try to guess if we're installing on SUSE 11, as it needs a different flow to work
+  if cat /etc/SuSE-release 2>/dev/null | grep VERSION | grep 11; then
+    SUSE11="yes"
+  fi
+
   echo -e "\033[34m\n* Installing YUM Repository for Datadog\n\033[0m"
   $sudo_cmd sh -c "echo -e '[datadog]\nname=datadog\nenabled=1\nbaseurl=https://yum.${repo_url}/suse/${dd_agent_dist_channel}/${dd_agent_major_version}/${ARCHI}\ntype=rpm-md\ngpgcheck=1\nrepo_gpgcheck=0\ngpgkey=https://yum.${repo_url}/DATADOG_RPM_KEY.public\n       https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public' > /etc/zypp/repos.d/datadog.repo"
 
   echo -e "\033[34m\n* Importing the Datadog GPG Keys\n\033[0m"
-  $sudo_cmd rpm --import https://yum.${repo_url}/DATADOG_RPM_KEY.public
-  $sudo_cmd rpm --import https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public
+  if [ "$SUSE11" == "yes" ]; then
+    $sudo_cmd curl -o /tmp/DATADOG_RPM_KEY.public https://yum.${repo_url}/DATADOG_RPM_KEY.public
+    $sudo_cmd rpm --import /tmp/DATADOG_RPM_KEY.public
+    $sudo_cmd curl -o /tmp/DATADOG_RPM_KEY_E09422B3.public https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public
+    $sudo_cmd rpm --import /tmp/DATADOG_RPM_KEY_E09422B3.public
+  else
+    $sudo_cmd rpm --import https://yum.${repo_url}/DATADOG_RPM_KEY.public
+    $sudo_cmd rpm --import https://yum.${repo_url}/DATADOG_RPM_KEY_E09422B3.public
```

Comment:
We could import only the new key when installing A7. This would serve to check we have signed it with the correct key. We can do this to a different PR